xiaqianZhang/assignments/cyclicSort/cyclicSortEasy/cyclicSortEasy.py

```diff
@@ -0,0 +1,99 @@
+# Problem Statement #
+
+# We are given an array containing ‘n’ objects. Each object, when created, 
+# was assigned a unique number from 1 to ‘n’ based on their creation sequence. 
+# This means that the object with sequence number ‘3’ was created just before the object with sequence number ‘4’.
+
+# Write a function to sort the objects in-place on their creation sequence number in O(n) and without any extra space. 
+# For simplicity, let’s assume we are passed an integer array containing only the sequence numbers, 
+# though each number is actually an object.
+
+# input: unsorted list of object that is in incrementing order by 1, start from 1, ex: 1, 2, 3,...6
+# output: sorted list
+# edges case: if the len of the list <2: return the list
```

Comment:
Love that you wrote an edge case, but don't forget to write a test case to match as well!